app/src/main/java/com/vero/tasky/auth/presentation/components/AuthHeaderText.kt

```diff
@@ -0,0 +1,18 @@
+package com.vero.tasky.auth.presentation.components
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.vero.tasky.ui.theme.headerText
+
+@Composable
+fun AuthHeaderText(
```

Comment:
On every composable, I like to pass a modifier parameter
Even if you never end up using it, it'll allow other screens to modify the composable without modifying the code inside of it

So here: 
```
fun AuthHeaderText(
 @StringRes titleRes: Int,
modifier: Modifier = Modifier
)